src/test/kotlin/org/rust/lang/ProjectDescriptor.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang
+
+import com.intellij.testFramework.LightProjectDescriptor
+import kotlin.reflect.KClass
+
+/**
+ * Allows to set [LightProjectDescriptor] for a specific test.
+ * The [descriptor] class must be a kotlin object (`object Foo : LightProjectDescriptor`).
+ *
+ * Example values:
+ * - [WithStdlibRustProjectDescriptor]
+ * - [WithStdlibAndDependencyRustProjectDescriptor]
+ *
+ * @see RsTestBase.getProjectDescriptor
+ */
+@Target(AnnotationTarget.FUNCTION, AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class ProjectDescriptor(val descriptor: KClass<out LightProjectDescriptor>)
```

Comment:
If we want to use only object classes as descriptors, maybe we pass object instance here directly instead of `KClass`?

Reply:
It isn't possible for java/kotlin annotations =(